scenes/enumerated/problem_12649/armenian_text.py

```diff
@@ -0,0 +1 @@
+taskNumberString = r"$\textrm{Խ.} 12649$"
```

Comment:
It would be nice, if you added a new line symbol at the end of the file.

Reply:
ի դեպ, մինչև էս ոնցոր բոլոր դեպքերում վարժության համարը տեխով չի գրած